src/index.js

```diff
@@ -131,8 +131,8 @@ class Service {
 
     let { query, options } = multiOptions(id, params, this.id);
 
-    // We can not update the id
-    delete data[this.id];
```

Comment:
I don't think we really defined how other adapters behave with custom ids. Here we definitely do have to add it back as the resource would be orphaned otherwise and I don't think you should be able change an id (although we could check for `if(!data[this.id])`). We should probably add the same logic in `patch`.